docs/install/proxmox/proxmox-setup.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Installing Colony with Promox
+description: Follow the steps to install Colony with Proxmox
+sidebar_position: 2
+---
+
+## Installing Colony with Proxmox
+
+This guide walks through the setup of Laptop-0, a machine for running Colony, and 3 additional VM's for Colony to auto discovery and install a K3S cluster.  Some items to note:
+
+- Colony requires Docker.
+- If you have a firewall on your host machine (laptop 0) it needs to allow the following ports (ports TBA).
+
+Warning: Colony will be able to PXE boot all machines in your network.
+
+## Getting Started
+
+The steps here are optional, but we recommend completing these to create a Proxmox template VM that you can clone to for better performance and asset creation.
```

Comment:
```suggestion
The steps here are optional, but we recommend completing these to create a Proxmox template VM that you can clone to quickly create virtual machines to experiment with. You can move on *add link here* if you have an existing virtual machine that you want to use as Laptop-0. 
```

Reply:
What link are you suggesting we add here?